.vscode/launch.json

```diff
@@ -19,15 +19,15 @@
       "type": "python",
       "request": "launch",
       "program": "${workspaceFolder}/python/vTestInterface.py",
-      "cwd": "C:/RDS/VectorCAST/FOO",
+      "cwd": "C:/RDS/VectorCAST/CodeBasedTests/unitTests",
```

Comment:
@johnpaliotta happy to discuss, but I think we should start rejecting PRs that change this file.

This feel seems to get changed in every PR (I understand why), but I think we should start not including changes to this file inside of "functional" changes to the extension.

Reply:
Let's just remove it from the repo

Reply:
How about we clean it up, and move it to `.vscode/launch.json.example`? Then you can copy/make new copies as you like, but they won't be tracked.

To be clear: I see value in this file being in the repo, but I'm trying to avoid the "noise".

Reply:
Sure, it saves me a ton of time to have it match my setup